packages/material-ui-lab/src/SpeedDial/SpeedDial.test.js

```diff
@@ -339,7 +335,10 @@ describe('<SpeedDial />', () => {
       });
     });
 
-    describe('actions navigation', () => {
+    // eslint-disable-next-line func-names
+    describe('actions navigation', function() {
+      this.timeout(5000); // This tests are really slow.
```

Comment:
Why do we need the timeout?

Reply:
I have to restart 6 builds this morning to get 4 merge commits green.

![Capture_d’écran_2019-04-21_à_23 00 28](https://user-images.githubusercontent.com/3165635/56501618-b52c3980-650f-11e9-923d-0b8a74322413.png)

Reply:
Again: https://circleci.com/gh/mui-org/material-ui/87995?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link.